app/controllers/clients_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ClientsController < ApplicationController
+  before_action :find_client, only: [:show, :destroy, :edit, :update]
```

Comment:
be consistent here, use set_client as method name....and similar across other controllers, as before_action.

Reply:
Fixed.